aio/aio-proxy/aio_proxy/doc/open-api.yml

```diff
@@ -1,334 +1,373 @@
-  version: 1.0.0
+  version: 2.0.0
   contact:
     email: recherche-entreprises@api.gouv.fr
     name: Contact API Recherche d’entreprises
     url: https://recherche-entreprises.api.gouv.fr/
   description: >
     # Bienvenue sur la documentation interactive d'API Recherche d’entreprises !
-    
-    L’API Recherche d’entreprises permet à tout le monde de rechercher et de trouver 
-    une entreprise française par sa dénomination, ou son adresse.
-    
+
+    L’API Recherche d’entreprises permet à tout le monde de rechercher et de
+    trouver  une entreprise française par une dénomination et/ou une adresse.
+
     ## Données accessibles dans l'API
 
-    L’API étant totalement ouverte d'accès, elle comporte des limitations. Ainsi ne sont pas accessibles dans l'API :
+    L’API étant totalement ouverte d'accès, elle comporte des limitations. Ainsi
+    ne sont pas accessibles dans l'API :
 
     - les prédécesseurs et successeurs d'un établissement
-    
+
     - les entreprises non-diffusibles
-    
+
     - les entreprises qui se sont vues refuser leurs immatriculation au RCS
-    
-    
-    **Attention :** cette API ne permet pas d'accèder aux données complètes de  la 
-    base Sirene, mais uniquement de rechercher une entreprise, par sa dénomination ou
-    son adresse. Pour savoir comment obtenir les données complètes,
-    consultez [notre fiche explicative.](https://api.gouv.fr/guides/quelle-api-sirene)
-    
+
+
+    **Attention :** cette API ne permet pas d'accèder aux données complètes de 
+    la  base Sirene, mais uniquement de rechercher une entreprise, par sa
+    dénomination ou son adresse. Pour savoir comment obtenir les données
+    complètes, consultez [notre fiche
+    explicative.](https://api.gouv.fr/guides/quelle-api-sirene)
+
 
     ## Commencer à utiliser l'API
-    
-    L'API est accessible à partir de cette adresse : [https://recherche-entreprises.api.gouv.fr](https://recherche-entreprises.api.gouv.fr)
-    
-    
+
+    L'API est accessible à partir de cette adresse :
+    [https://recherche-entreprises.api.gouv.fr](https://recherche-entreprises.api
+    .gouv.fr)
+
+
     **Attention :** Vous devez rajouter votre requête sur l’adresse.
 
     ## Limite des requêtes
-    
-      Le serveur accepte un maximum de 7 requêtes par seconde. Au delà, un code 429 est renvoyé indiquant que la volumétrie d'appels a été dépassée. 
-  
-  
-    
-    ## Monitoring de l'API
-    
-    Une supervision du service en temps réel est disponible à cette adresse : [https://stats.uptimerobot.com/kXzwzfk7BE](https://stats.uptimerobot.com/kXzwzfk7BE)
 
+      Le serveur accepte un maximum de 7 requêtes par seconde. Au delà, un code 429 
+    est renvoyé indiquant que la volumétrie d'appels a été dépassée. 
 
 
+
+    ## Monitoring de l'API
+
+    Une supervision du service en temps réel est disponible à cette adresse :
+    [https://stats.uptimerobot.com/kXzwzfk7BE](https://stats.uptimerobot.com/kXzwzfk7BE)
```

Comment:
on veut pas plutôt lier vers : https://annuaire-entreprises.data.gouv.fr/sources-de-donnees#APIRecherched%E2%80%99entreprises ?

Reply:
You're right 🥁